### ISSUE TITLE
Duplicate symlinks that reside within the dotfiles repository

### ISSUE THREAD

**Issue by dpc** [dylanmckay/polk#7]
So I've switched to `polk` from `stow`. (Yay!)

I've decided to put the symlink `~/dot -> ~/.polk/cache/dpc/dotfiles` inside the dotfiles itself.

```
dpc@futex ~/.p/u/d/dotfiles (master)> ls -alh
...
lrwxrwxrwx. 1 dpc dpc   27 Aug  9 20:33 dot -> ../.polk/users/dpc/dotfiles
...
```

But now I see that `polk` is skiping symlinks inside the directory:

https://github.com/dylanmckay/polk/blob/48d22c9c6fcf97f6bf0ea72e84e7e944bf865044/src/cache.rs#L232

I wonder if could be supported? GNU Stow was working fine with symlinks. All I want is for `polk` to create a symlink that points exactly the same way as the original one.

**Comment by dylanmckay** [comment]
It sounds like an easy enough feature - what would _you_ expect to happen in this case? We make a new symlink to the same destination, or we create a symlink to the symlink which in turn links to the original destination?

**Comment by dpc** [comment]
A new symlink should appear that is exactly the same. `dot -> ../.polk/users/dpc/dotfiles`. If it was relative, it should stay relative. If it was absolute, it should stay absolute.

**Comment by dpc** [comment]
Can you release? :)

**Comment by dylanmckay** [comment]
Looks like git2-rs has released a new version, so I should be able to now :)

**Comment by dylanmckay** [comment]
Just released [`1.1.2`](https://github.com/dylanmckay/polk/releases/tag/1.1.2)

**Comment by dylanmckay** [comment]
Closing this issue now cause the current version of polk creates a symlink at `~/.dot`.
